ci/Jenkinsfile

```diff
@@ -70,8 +70,9 @@ pipeline {
     TESTRAIL_URL = 'https://ethstatus.testrail.net'
     TESTRAIL_PROJECT_ID = 17
 
+
     /* Runtime flag to make testing of the app easier. */
-    STATUS_RUNTIME_TEST_MODE = '1'
+    /* STATUS_RUNTIME_TEST_MODE = '1' */
```

Comment:
If you're commenting something out like this add a comment why it's commented out.

Reply:
Yes, correct, i added a reason of that to line 73 @jakubgs